code/ui-utils/src/main/java/com/adobe/marketing/mobile/utils/UiImageUtils.kt

```diff
@@ -59,8 +63,13 @@ internal object PushTemplateImageUtils {
      * @param urlList [String] containing an image asset url
      * @return [Int] number of images that were found in cache or successfully downloaded
      */
-    internal fun cacheImages(
-        urlList: List<String?>
+    fun cacheImages(
+        urlList: List<String?>,
+        downloadTimeoutInSeconds: Int = DEFAULT_DOWNLOAD_TIMEOUT_SECS,
+        bitmapQuality: Int = DEFAULT_BITMAP_QUALITY,
+        bitmapWidth: Float = CAROUSEL_MAX_BITMAP_WIDTH.toFloat(),
+        bitmapHeight: Float = CAROUSEL_MAX_BITMAP_HEIGHT.toFloat(),
```

Comment:
As discussed offline, lets not use these are default values for `bitmapWidth` and `bitmapHeight`. Each UI solution might have its own requirements